sql/2020/19_Compression/distribution of text compression lighthouse

```diff
@@ -0,0 +1,15 @@
+#standardSQL
```

Comment:
Please rename these files to use underscore_notation and end with the `.sql` file extension